pdr_backend/lake/test/test_etl.py

```diff
@@ -222,21 +217,31 @@ def test_move_from_temp_tables_to_live(setup_data):
     assert len(gql_tables) == 5
 
     dummy_schema = {"test_column": str}
-    pds.insert_to_table(pl.DataFrame([], schema=dummy_schema), "_temp_a")
-    pds.insert_to_table(pl.DataFrame([], schema=dummy_schema), "_temp_b")
-    pds.insert_to_table(pl.DataFrame([], schema=dummy_schema), "_temp_c")
+
+    # Insert temp ETL tables w/ dummy data into DuckDB
+    temp_bronze_table_names = []
+    for table_name in etl.bronze_table_names:
+        pds.insert_to_table(
+            pl.DataFrame([], schema=dummy_schema), TempTable(table_name).fullname
+        )
+        temp_bronze_table_names.append(TempTable(table_name).fullname)
 
     # check if tables are created
     table_names = pds.get_table_names()
-    etl.temp_table_names = ["a", "b", "c"]
+
+    # Assert all temp bronze table names exist in table_names
+    assert all(
+        table in table_names for table in temp_bronze_table_names
+    ), "Not all temporary bronze tables were created successfully"
+
     etl._move_from_temp_tables_to_live()
 
-    # check "c" exists in permanent tables
+    # check all temp tables are dropped, and regular bronze_tables exist
     table_names = pds.get_table_names()
-    assert len(table_names) == 8
-    assert "c" in table_names
-    assert "a" in table_names
-    assert "b" in table_names
+    assert all(
+        table in table_names for table in etl.bronze_table_names
+    ), "Not all temporary bronze tables were moved to live tables successfully"
+    assert len(table_names) == 6
```

Comment:
It's not every table that gets moved ("a", "b", "c"), it's the bronze temp tables for the ETL that do.